src/clashboard/clinical_trials_data.py

```diff
@@ -1,9 +1,16 @@
 
 
-class ClashInterface:
+class ClinicalTrialsData:
 
-    def __init__(self, db):
-        pass
+    studies = None
+    db = None
+    conn = None
+    type_counts = None
+    curr_group = 'phase'
+
+    def __init__(self, db, conn):
```

Comment:
Since this will now be the production version of the class, it is ok to declare the database fields internally.  We should go back to an `__init__` with no parameters.